models/comment_thread.rb

```diff
@@ -73,6 +73,7 @@ class CommentThread < Content
   before_destroy :destroy_subscriptions
 
   scope :active_since, ->(from_time) { where(:last_activity_at => {:$gte => from_time}) }
+  scope :exclude_standalone, ->() { where(:context.ne => :standalone) }
```

Comment:
Is there any sort of "default scope" in Mongoid? It would be really nice if `exclude_standalone` were applied to all queries over `CommentThread`s by default, and `include_standalone` could be used for teams-specific stuff.